### ISSUE TITLE
Using your implementation on 1D data

### ISSUE THREAD

**Issue by erap129** [sash-a/CoDeepNEAT#1]
Hello,  
I am currently developing a NAS algorithm of my own - [https://bitbucket.org/eladrapaport/eegnas/src/master/](url) and I wanted to ask:  
1. I see this code is being actively developed, do you think it is in a state where I can clone it and use it as a benchmark against my algorithm?  
2. If so, can you explain a bit about the outputs generated by your algorithm? And the overall required changes you think supporting 1D data will entail. I have no problem contributing my code once I successfully adapt your code to 1D time series data.  
  
Thank you very much.

**Comment by sash-a** [comment]
It can certainly be used a benchmark, to get the best config options see our papers. We are currently doing quite a big rewrite, so if you can wait a month it will be much easier to work with.

As for time series data original CoDeepNEAT does work with that, but our implementation is currently only geared towards image data and it also does not yet support LSTM layers. To allow for 1D data have a look at the dataloader class in src/validator and you would probably need to change the output sizes the networks expect and the loss functions they use (see src/NeuralNetwork/ModuleNet).

But again if possible I would strongly reccomend you wait a month since our rewrite is geared to making this a lot easier to use. But if you need it now, just check the readme for previous semi-stable commit and go from there.
